feeds.conf.default

```diff
@@ -1,4 +1,4 @@
-src-git packages https://github.com/openwrt/packages.git
+src-git packages https://github.com/CreatorDev/packages.git
```

Comment:
The master branch just tracks upstream so this essentially does nothing. You need to point it to the creator branch.